test/validation/api/traffic_mngr/traffic_mngr.c

```diff
@@ -4180,26 +4248,45 @@ static void traffic_mngr_test_tm_create(void)
 
 static void traffic_mngr_test_shaper(void)
 {
-	CU_ASSERT(!odp_cunit_ret(test_shaper_bw("bw1",
-						"node_1_1_1",
-						0,
-						MBPS * 1)));
-	CU_ASSERT(!odp_cunit_ret(test_shaper_bw("bw4",
-						"node_1_1_1",
-						1,
-						4   * MBPS)));
-	CU_ASSERT(!odp_cunit_ret(test_shaper_bw("bw10",
-						"node_1_1_1",
-						2,
-						10  * MBPS)));
-	CU_ASSERT(!odp_cunit_ret(test_shaper_bw("bw40",
-						"node_1_1_1",
-						3,
-						40  * MBPS)));
-	CU_ASSERT(!odp_cunit_ret(test_shaper_bw("bw100",
-						"node_1_1_2",
-						0,
-						100 * MBPS)));
+	if ((tm_shaper_min_rate <= 1 * MBPS) &&
+	    (tm_shaper_max_rate >= 1 * MBPS)) {
+		CU_ASSERT(!odp_cunit_ret(test_shaper_bw("bw1",
```

Comment:
The rate checks is ok, but I wonder how much test_shaper_bw() depends on burst size being 10000 and not some clamped value?